serverless.yml

```diff
@@ -19,12 +19,13 @@ provider:
 functions:
   placement:
     handler: kinesis/placement.handler
+    timeout: 60
     events:
       - stream:
           arn: arn:aws:kinesis:us-east-1:056154071827:stream/snowplow_enrich
           batchSize: 100
           startingPosition: LATEST
-          enabled: false
+          enabled: true
```

Comment:
Does this enable it on local?

Reply:
It enables it by default in AWS. I had it disabled and would manually enable the kinesis stream to test. I think it's ready to just start using when we deploy.
https://serverless.com/framework/docs/providers/aws/events/streams/